README.md

```diff
@@ -40,16 +40,23 @@ play_routes_pinned_maven_install()
 
 This installs `rules_play_routes` to your `WORKSPACE` at the specified commit. Update the commit as needed.
 
-## Stardoc Documentation
-http://lucidsoftware.github.io/rules_play_routes/
+## Third Party Dependencies
+We use [rules_jvm_external](https://github.com/bazelbuild/rules_jvm_external) to import third party dependencies. You can change dependencies in `maven_install` in `workspace.bzl` for main `rules_play_routes` implementation or `test_workspace.bzl` for testing.
 
-Stardoc is replacing Skydoc and is currently under development. Doc is likely going to look funny for a while.
+Run the script manually to update the corresponding `.json`. You will never want to change the `.json` directly.
+```bash
+scripts/gen-deps.sh
+```
 
-### Updating Stardoc
-Stardoc is automatically updated on build merged into the master branch. To update the documentation, please submit a pull request. The doc will be updated when it is merged.
+## Stardoc Documentation
+* Simple Core API
```

Comment:
Since we only have one rule, Add this in the Overview section:

>Simple Core API: [play_routes](docs/stardoc/play-routes.md)

Reply:
Done.